ui/raidboss/data/07-dt/raid/r4s.ts

```diff
@@ -275,6 +383,84 @@ export interface Data extends RaidbossData {
 const triggerSet: TriggerSet<Data> = {
   id: 'AacLightHeavyweightM4Savage',
   zoneId: ZoneId.AacLightHeavyweightM4Savage,
+  config: [
+    {
+      id: 'ionCluster',
+      name: {
+        en: 'Ion Cluster Debuff Strategy',
+      },
+      comment: {
+        en: `Strategy for resolving debuffs during Ion Cluster.
+
+             None - Call the debuff only, no strategy.
+             DN - use rivet positions based on the shabin pastebin.`,
+      },
+      type: 'select',
+      options: {
+        en: {
+          'None': 'none',
+          'DN': 'DN',
+        },
+      },
+      default: 'none',
+    },
+    {
+      id: 'witchHunt',
+      name: {
+        en: 'Witch Hunt Bait Strategy',
+      },
+      comment: {
+        en: `Strategy for baiting Witch Hunt AoEs.
+
+             None - Call both party and bait positions, no strategy.
+             DN - Flexible priority to maximize melee uptime, with tanks taking
+                  first near bait, healers taking first far bait, melees taking
+                  second near bait, and ranged taking second far bait.`,
```

Comment:
I'm not sure if the whitespace will get properly translated/formatted in HTML.  Is there an existing trigger where we do something similar that we can check to confirm proper UI display?

Reply:
It does not, I checked. I'll see if I can come up with a better thing. Perhaps I can use html breaks to separate each strategy?